### ISSUE TITLE
 REST Fest Videos Project API/apps

### ISSUE THREAD

**Issue by BigBlueHat** [RESTFest/2016-Greenville#2]
The [REST Fest Videos Project](http://videos.restfest.org/) is currently a collection of several Vimeo channels. To find videos, you have to search all of Vimeo. Descriptions are generally basically informative (thanks videography people!), but the whole thing would be most amazing with a People, Topic, Video, Year, etc. API...dontcha think?!

**Comment by timdietrich** [comment]
I agree - an API would be awesome! And if there were one, I'd be happy to create a RESTFest Apple TV app that would pull the video info from the API, and serve up the videos from Vimeo. Seems like this might make for an interesting Hack Day project.

**Comment by arno-di-loreto** [comment]
Great idea!
